addon/utils/validators/ember-component.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * The PropTypes.EmberComponent validator
+ */
+
+import Ember from 'ember'
+const {typeOf} = Ember
+
+import logger from '../logger'
+
+export default function (ctx, name, value, def, logErrors, throwErrors) {
+  const isObject = typeOf(value) === 'object'
+
+  const valid = isObject && Object.keys(value).some((key) => {
+    return key.indexOf('COMPONENT_CELL') === 0
```

Comment:
Can we flag this in some was as "internal", just in case things break we'll know where to look first